### ISSUE TITLE
Contributing..

### ISSUE THREAD

**Issue by chanq-io** [ekzhang/composing.studio#25]
Hey! I’m just reaching out to see if I can contribute to this project. I’ve worked in the music tech space for 8 years, primarily with Python / C++ / JS, but I’ve recently been learning rust and am looking for open source projects to contribute to. Let me know if you’re happy for me to get involved and whether you’ve got a roadmap/backlog I can pick up tasks from

**Comment by nrgapple** [comment]
I agree! this is one of the coolest examples of wasm in practice! I'd love to learn more about what is happening between the rust code and ts

**Comment by wxtim** [comment]
I too am happy to help, if I can, although my primary expertise is in Python, Bash and esoteric weather encodings I'm supposed to be learning some JS. I have an ABC Sphinx Ext at an embryonic stage.

**Comment by chanq-io** [comment]
@ekzhang any thoughts on how we can get involved?

**Comment by ekzhang** [comment]
> Hey! I’m just reaching out to see if I can contribute to this project. I’ve worked in the music tech space for 8 years, primarily with Python / C++ / JS, but I’ve recently been learning rust and am looking for open source projects to contribute to. Let me know if you’re happy for me to get involved and whether you’ve got a roadmap/backlog I can pick up tasks from

Thanks for your interest @chanq-io! It is very appreciated. I think all of us are a little bit busy at the moment, which is why we haven't been able to get back to you.

There was a recent crash bug that was reported, and if you would like, you could look at that if you'd like? It would be appreciated! We don't have any immediate plans to add more features to the app right now, but even more so, we'd love to hear your feedback or ideas, especially as someone who has worked in the music tech space.



> I agree! this is one of the coolest examples of wasm in practice! I'd love to learn more about what is happening between the rust code and ts

Rust is used on the backend for the operational transformation algorithm and WebSocket server (using Rustpad), and the TypeScript code is used to build the frontend web interface + user interface / editor / sheet music display.
